parastell/magnet_coils.py

```diff
@@ -5,10 +5,15 @@
 import numpy as np
 
 import cadquery as cq
-import cubit
 
 from . import log
-from . import cubit_io as cubit_io
+from .cubit_utils import (
+    import_geom_to_cubit,
+    export_mesh_cubit,
+    mesh_volume_skeleton,
+    get_last_id,
+)
+from . import cubit_utils
```

Comment:
I'd like to note (for reviewers) that `cubit_utils` is imported separately here for its `initialized` variable. If `initialized` is imported into the namespace of `magnet_coils`, changes to the variable do not persist when modified by calls to the imported functions

Reply:
Should that be added as a comment?